action.yaml

```diff
@@ -242,11 +242,14 @@ runs:
 
           if [[ "${{ inputs.preview-name }}" != "" ]]; then
             BRANCH_DEPLOYMENT_NAME="${{ inputs.preview-name }}"
-
-          else
-            DEPLOYMENT_NAME="$(astro deployment inspect $DEPLOYMENT_ID --clean-output --key configuration.name)"
-            BRANCH_DEPLOYMENT_NAME=${{ github.event.ref }}_$DEPLOYMENT_NAME
-            BRANCH_DEPLOYMENT_NAME="${BRANCH_DEPLOYMENT_NAME// /_}"
+            else
+              # Get deployment name from Astro if preview-name is not provided
+              DEPLOYMENT_NAME="$(astro deployment inspect $DEPLOYMENT_ID --clean-output --key configuration.name)"
+              # using github.event.ref
+              branch_ref="${{ github.event.ref }}"
+              branch_name="${branch_ref##*/}"
+              BRANCH_DEPLOYMENT_NAME=${branch_name}_$DEPLOYMENT_NAME
+              BRANCH_DEPLOYMENT_NAME="${BRANCH_DEPLOYMENT_NAME// /_}"
```

Comment:
looking at indentation, I think we need to fix that.

Also, I think we should keep the logic consistent, i.e. carry the same branch name logic from the one above.
```
            if [[ ${GITHUB_HEAD_REF##*/} != "" ]]; then
              BRANCH_DEPLOYMENT_NAME=${GITHUB_HEAD_REF##*/}_$DEPLOYMENT_NAME
            else
              branch_ref="${{ github.ref }}"
              branch_name="${branch_ref##*/}"
              BRANCH_DEPLOYMENT_NAME=${branch_name}_$DEPLOYMENT_NAME
            fi
            BRANCH_DEPLOYMENT_NAME="${BRANCH_DEPLOYMENT_NAME// /_}"
            echo $BRANCH_DEPLOYMENT_NAME
```

Reply:
I tried using the same logic, but since this involves deleting a branch, GITHUB_HEAD_REF refers to a different branch than the one being deleted.

Indentation got fixed.